### ISSUE TITLE
[FIXED] - Failed to parse source map - @metamask/detect-provider/src/index.ts

### ISSUE THREAD

**Issue by nandorcsupor** [PatrickAlphaC/defi-stake-yield-brownie#7]
Could you please help me with this ?  Anyone had the same issue ? 

WARNING in ./node_modules/@metamask/detect-provider/dist/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/home/nandor/Solidity/defi-stake-yield-brownie/front_end/node_modules/@metamask/detect-provider/src/index.ts' file: Error: ENOENT: no such file or directory, open '/home/nandor/Solidity/defi-stake-yield-brownie/front_end/node_modules/@metamask/detect-provider/src/index.ts'
 @ ./node_modules/@usedapp/core/dist/esm/src/helpers/injectedProvider.js 2:0-63 4:33-55
 @ ./node_modules/@usedapp/core/dist/esm/src/providers/network/injectedNetwork/provider.js 4:0-72 19:4-23
 @ ./node_modules/@usedapp/core/dist/esm/src/providers/network/injectedNetwork/index.js 2:0-27 2:0-27
 @ ./node_modules/@usedapp/core/dist/esm/src/providers/index.js 6:0-42 6:0-42
 @ ./node_modules/@usedapp/core/dist/esm/src/index.js 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526 3:0-526
 @ ./src/App.tsx 4:0-54 9:30-42 11:24-37 11:39-54
 @ ./src/index.tsx 6:0-24 10:33-36

 - Does not render anything when I run yarn start

**Comment by nandorcsupor** [comment]
I just made an "index.ts" in my "/src" folder that only contains an empty export {}. Now the error is gone. But how do I get the page the page to load that is is the "App.tsx" ? (I still have the original "index.tsx" file which I have not modified. Why wouldn't it just render that index.tsx like before ?

**Comment by nandorcsupor** [comment]
When I modify the "@usedapp/core": version I'm able to get rid of the error. But still doesn't render anything.
Also same thing with using npm instead of yarn all the way.

**Comment by nandorcsupor** [comment]
Fixed. Sorry for spamming the "Issues" tab. But it might be important for someone else in the future.
I fixed it by using exactly(!) the same imports both for material-ui and for usedapp/core as Patrick did in this project. These are : 
- "@material-ui/core": "^4.11.4",
- "@material-ui/lab": "^4.0.0-alpha.58",
- "@usedapp/core": "^0.5.4",

Seems like these 2 imports do not like each other in future releases.
